packages/components/src/organisms/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -56,17 +69,22 @@ function ImageGallerySelector({ images, onSelect, currentImageIdx }: Props) {
   return (
     <section
       data-fs-image-gallery-selector
-      className={styles.fsImageGallerySelector}
+      data-testid={testId}
       aria-roledescription="carousel"
       aria-label="Product images"
     >
       {elementHasScroll && !firstImageInView && (
-        <UIIconButton
-          data-fs-image-gallery-selector-control-button
-          aria-label="backward slide image selector"
-          icon={<Icon name="ArrowLeft" width={24} height={24} />}
-          onClick={() => moveScroll(elementsRef.current, -SCROLL_MARGIN_VALUE)}
-        />
+        <div data-fs-image-gallery-selector-control>
+          <IconButton
+            data-fs-image-gallery-selector-control-button
+            aria-label="backward slide image selector"
```

Comment:
Same here, receive this label as prop, and set this value as default

Reply:
I don't think we need to replicate for this case and the [other one](https://github.com/vtex/faststore/pull/1681#discussion_r1154816037) you mentioned, because those are standard navigation behaviors, we can leave those set to guarantee the a11y for the navigation controls, what do you think? cc @hellofanny